stations.csv

```diff
@@ -17019,7 +17019,7 @@ id;name;slug;uic;uic8_sncf;longitude;latitude;parent_station_id;is_city;country;
 18552;Locarno Piazza Stazione;locarno-piazza-stazione;;;8.8003760000;46.1724900000;18295;f;CH;f;Europe/Zurich;f;;f;;f;8594370;t;;f;;f;;f;;f;;;;;;;;;;;;;;;;;;
 18553;Zermatt Bahnhof;zermatt-bahnhof;;;7.7484580000;46.0241940000;;f;CH;f;Europe/Zurich;f;;f;;f;8594556;t;;f;;f;;f;;f;;;;;;;;;;;;;;;;;;
 18554;Sierre Hôtel de Ville - CFF;sierre-hotel-de-ville-cff;;;7.5339580000;46.2933400000;;f;CH;f;Europe/Zurich;f;;f;;f;8594615;t;;f;;f;;f;;f;;;;;;;;;;;;;;;;;;
-18555;Zürich;zurich;;;8.5405340000;47.3789680000;;f;CH;f;Europe/Zurich;f;;f;;f;;f;;f;;f;;f;;f;;;;;;;;;;;;;;;;;;
+18555;;;;;;;;f;CH;f;Europe/Zurich;f;;f;;f;;f;;f;;f;;f;;f;;;;;;;;;;;;;;;;;;
```

Comment:
There are plenty of ghost stations scattered around the file. We leave the
country and time zones because of (legacy?) db constraints.

We could probably part with them provided no sale occured.

Nobody in their own mind would reuse the ids anyway.